packages/client/modules/userDashboard/components/InvoiceRow/InvoiceRow.tsx

```diff
@@ -43,18 +43,17 @@ const InfoRowRight = styled('div')({
   textAlign: 'right'
 })
 
-const RowLink = styled('div')({
+const LinkStyles = styled('div')({
   color: PALETTE.SLATE_700,
   alignItems: 'flex-start',
   display: 'flex',
   justifyContent: 'space-between',
   textDecoration: 'none',
-  width: '100%',
-  '&:hover': {
-    cursor: 'pointer'
-  }
+  width: '100%'
 })
 
+const RowLink = LinkStyles.withComponent(Link)
```

Comment:
-1 Does not work, it redirects to
`http://localhost:3000/me/organizations/oU3Kik4zU4/https://invoice.stripe.com/...` for me

Reply:
🤦‍♂️ I've replaced the RowLink element with an achor element using tailwind. It's fixed for me now!